### PR TITLE
Retrieve the release_version from the notes assets path

### PR DESCRIPTION
**Pull request by saschagrunert**
This allows us to get rid of the `release_version` field in k/release
which has been only used for this website and does not provide any
additional value.

### CODE REVIEW

**Comment by saschagrunert** [comment]
:green_circle: tests

File: src/app/notes/notes.service.ts [hunk 1]
```diff
@@ -35,15 +41,30 @@ export class NotesService {
...
     return list;
   }
+
+  /**
+   * Retrieve the release version from a asset path
+   *
+   * @returns The release version
+   */
+  private releaseVersionFromPath(path: string): string {
+    return path.replace(/^.*release-notes-/, '').replace(/.json$/, '');
```
Comment (puerco) [inline src/app/notes/notes.service.ts:68]: :eye_speech_bubble: I think we should note in a manual somewhere that filenames now _MUST_  (in 
 an RFC way) conform to this pattern.

Comment (saschagrunert) [reply src/app/notes/notes.service.ts:68]: The tooling ensures that, right? :)

**Pull request merged**